interfaces/core.go

```diff
@@ -25,56 +25,56 @@ import (
 	"regexp"
 )
 
-// Skill represents a capacity offered by a snap.
-type Skill struct {
-	Name  string
-	Snap  string
-	Type  string
-	Attrs map[string]interface{}
-	Apps  []string
-	Label string
+// Plug represents a capacity offered by a snap.
+type Plug struct {
+	Name      string
+	Snap      string
+	Interface string
+	Attrs     map[string]interface{}
+	Apps      []string
+	Label     string
 }
 
-// Slot represents the potential of a given snap to use a skill.
+// Slot represents the potential of a given snap to connect to a plug.
 type Slot struct {
-	Name  string
-	Snap  string
-	Type  string
-	Attrs map[string]interface{}
-	Apps  []string
-	Label string
+	Name      string
+	Snap      string
+	Interface string
+	Attrs     map[string]interface{}
+	Apps      []string
+	Label     string
 }
 
-// Type describes a group of interchangeable capabilities with common features.
-// Types are managed centrally and act as a contract between system builders,
+// Interface describes a group of interchangeable capabilities with common features.
+// Interfaces are managed centrally and act as a contract between system builders,
```

Comment:
s/are managed centrally and//

That became invalid once we expanded the scope moving out of "interface type". People will manage interfaces on their own snappy systems.